test/BasePaymentTest.php

```diff
@@ -199,6 +199,28 @@ public function createBasket(): Basket
         return $basket;
     }
 
+    /**
+     * Creates a v2 Basket resource and returns it.
+     *
+     * @return Basket
+     */
+    public function createV2Basket(): Basket
+    {
+        $orderId = 'b' . self::generateRandomId();
+        $basket = new Basket($orderId);
+        $basket->setTotalValueGross(99.99)
```

Comment:
I could not find a test which proves that basket v2 will be used, if old and new parameters are set.
Is this tested somehow?

Reply:
now i found it